pkg/filebuffer/filebuffer.go

```diff
@@ -41,12 +41,14 @@ type FileBuffer struct {
 	offset    int
 	offsets   []int
 	sourceMap *llb.SourceMap
+	mu        *sync.Mutex
```

Comment:
nit: This doesn't need to be a pointer.

I'm actually not sure why `buf` is a pointer - that also seems unnecessary.

Reply:
cool, removed the ptr for mu and buf